pennylane/ops/qubit.py

```diff
@@ -989,7 +1016,10 @@ class CRX(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
-    generator = [np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]]), -1 / 2]
+    generator = [
+        np.array([[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]]),
+        -1 / 2,
+    ]
```

Comment:
Just curious: how come these formatting changes were applied? :thinking: Usually we'd go with ` black -l 100` (let me know though if this was the output it gave!)

Reply:
I initially ran black through emacs where I was doing the code editing - I just set it up recently and I don't think the configuration is the same for the one that runs straight from the command line. I eventually ran it manually via command line, but then maybe changes made from the emacs version previous one were not reverted. Shall I revert to the original?

Reply:
Running black again, with `black -l 100`, should revert the formatting changes :)

Reply:
Oh got ya! If it's not too much work, might be worth reverting just so that no new changes are introduced due to an alternative formatting

Reply:
Will do! :)

Reply:
Alright this has been reverted!